### PR TITLE
Bug/1798 delete not actual level

### PR DESCRIPTION
**Pull request by Goven112**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: EPlast/EPlast.Resources/Roles.cs [hunk 1]
```diff
@@ -77,5 +77,18 @@ public static class Roles
             Roles.Admin,
             Roles.GoverningBodySectorSecretary
         };
+
+        public static List<string> DeleteableListOfRoles = new List<string>
+        {
+
```
Comment (VSBalanchuk) [inline EPlast/EPlast.Resources/Roles.cs:83]: This empty line is not needed. 
So, please, remove it.

File: EPlast/EPlast.Resources/Roles.cs [hunk 2]
```diff
@@ -77,5 +77,18 @@ public static class Roles
...
+       
+            Roles.OkrugaSecretary,
+            Roles.OkrugaHeadDeputy,
+            Roles.OkrugaHead,
+            Roles.CitySecretary,
+            Roles.CityHeadDeputy,
+            Roles.CityHead
+        };
+
+
```
Comment (VSBalanchuk) [inline EPlast/EPlast.Resources/Roles.cs:92]: These empty lines are not needed. 
So, please, remove it.

File: EPlast/EPlast.Resources/Roles.cs (hunk 1 above)
Comment (Goven112) [reply EPlast/EPlast.Resources/Roles.cs:83]: okey sorry , will be do

File: EPlast/EPlast.Resources/Roles.cs (hunk 2 above)
Comment (Goven112) [reply EPlast/EPlast.Resources/Roles.cs]: done

File: EPlast/EPlast.Tests/Services/City/CityParticipantsServiceTests.cs [hunk 3]
```diff
@@ -276,6 +279,71 @@ public async Task AddFollowerAsync_ReturnsCityMembersDTO()
...
+            User user = new User()
+            {
+                Id = "1234", 
+            };
+              _userManager.Setup(x => x.GetUserIdAsync(user)).ReturnsAsync(user.Id);
+            //act 
+
+            await _cityParticipantsService.AddFollowerAsync(It.IsAny<int>(), user);
+            _userManager.Verify(x => x.RemoveFromRolesAsync(It.IsAny<User>(), It.IsAny<IEnumerable<string>>()), Times.Once);
+        }
```
Comment (VSBalanchuk) [inline EPlast/EPlast.Tests/Services/City/CityParticipantsServiceTests.cs:345]: You either add all comments (arrange\act\assert) or none of them.

File: EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs [hunk 4]
```diff
@@ -24,24 +23,26 @@ public class CityParticipantsService : ICityParticipantsService
...
 
         public CityParticipantsService(IRepositoryWrapper repositoryWrapper,
                                        IMapper mapper,
                                        UserManager<User> userManager,
                                        IAdminTypeService adminTypeService,
                                        IEmailSendingService emailSendingService,
-                                       IMediator mediator,
-                                       IEmailContentService emailContentService)
+                                       ICityService cityService,
+                                       IEmailContentService emailContentService)
```
Comment (VSBalanchuk) [inline EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs:36]: Remove those whitespaces.

File: EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs [hunk 5]
```diff
@@ -24,24 +23,26 @@ public class CityParticipantsService : ICityParticipantsService
...
         {
             _repositoryWrapper = repositoryWrapper;
             _mapper = mapper;
             _userManager = userManager;
             _adminTypeService = adminTypeService;
             _emailSendingService = emailSendingService;
-            _mediator = mediator;
+            _cityService = cityService;
             _emailContentService = emailContentService;
+
```
Comment (VSBalanchuk) [inline EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs:45]: Remove this empty line.

File: EPlast/EPlast.Tests/Services/City/CityParticipantsServiceTests.cs (hunk 3 above)
Comment (Goven112) [reply EPlast/EPlast.Tests/Services/City/CityParticipantsServiceTests.cs]: +

File: EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs (hunk 4 above)
Comment (Goven112) [reply EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs]: done

File: EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs (hunk 5 above)
Comment (Goven112) [reply EPlast/EPlast.BLL/Services/City/CityParticipantsService.cs]: done

**Review by DimaDiD — APPROVED** [review]

**Review by VSBalanchuk — APPROVED** [review]

**Pull request merged**